src/modules/dashboard/components/InputStorageWidget/InputStorageWidget.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+import * as yup from 'yup';
+
+import { ActionTypes } from '~redux/index';
+import { ActionForm } from '~core/Fields';
+
+import InputStorageWidgetForm from './InputStorageWidgetForm';
+
+export interface FormValues {
+  inputStorageLot?: string;
+  inputNewValue?: string;
+}
+
+const InputStorageWidget = () => {
+  const validationSchema = yup.object().shape({
+    inputStorageLot: yup.string().required(),
+    inputNewValue: yup.string().required(),
+  });
+
+  return (
+    <ActionForm
```

Comment:
Not all of this is going to be wrapped inside an `ActionForm`. 

Please merge `InputStorageWidget.tsx` and `InputStorageWidgetForm.tsx` into only one file as this will be easier